conanfile.py

```diff
@@ -67,43 +58,51 @@
 def is_running_virtual_env():
     return sys.prefix != sys.base_prefix
 
+required_conan_version = ">=2.0.5"
+
 class BasiliskConan(ConanFile):
     name = "Basilisk"
-    homepage = "http://hanspeterschaub.info/basilisk"
+    homepage = "https://avslab.github.io/basilisk/"
     f = open('docs/source/bskVersion.txt', 'r')
     version = f.read()
     f.close()
-    generators = "cmake_find_package_multi"
-    requires = "eigen/3.4.0"
+    # generators = "CMakeDeps"
     settings = "os", "compiler", "build_type", "arch"
     build_policy = "missing"
     license = "ISC"
 
-    options = {"generator": "ANY"}
-    default_options = {"generator": ""}
+    # Requirements
+    requires = [
+        "eigen/3.4.0",
+    ]
+    package_type = "shared-library"  # TODO: Confirm this is correct.
```

Comment:
I believe this is correct since Basilisk is effectively a ton of shared libraries loaded by python. The `python-require` option is for Conan build system python libraries, not for stuff that actually gets built, so it is not what we want.

Reply:
Thanks for confirming. I'll remove the comment.